src/Oro/Bundle/UserBundle/Entity/User.php

```diff
@@ -1723,4 +1730,20 @@ public function getOrganization()
     {
         return $this->organization;
     }
+
+    /**
+     * @return string|null
+     */
+    public function getGoogleId()
+    {
+        return $this->googleId;
+    }
+
+    /**
+     * @param string|null $googleId
+     */
+    public function setGoogleId($googleId = null)
```

Comment:
All the setters in this class has `return $this;`. Why this method has no it?
Please add short description what this medod do.